apps/docs/src/development/index.md

```diff
@@ -154,8 +154,24 @@ Be careful when overriding styles globally since it will affect **EVERY** compon
 
 :::
 
+### Browser Support
```

Comment:
Would prefer to move this to the bottom of the page and change the headline to h2 :)

Reply:
like it!